src/main/net/polydawn/mdm/MdmArgumentParser.java

```diff
@@ -63,6 +63,12 @@ public void onAttach(Argument arg) {}
 		Subparser parser_status = subparsers
 			.addParser("status")
 			.help("list dependencies managed by mdm, and their current status.");
+		parser_status
+			.addArgument("--name")
+			.help("get the status of a particular dependency by name.  If specified, only that dependency will be included in the report.");
+		parser_status
+			.addArgument("--format")
```

Comment:
--format feels a little misleading to me, like you can say --format "%version-%name" and it'll print out the status information formatted to your specification. This feels more like --type to me. I know, it's not a big deal, but the clearer it is what a flag does, the easier it will be for people to use.